provider/README.md

```diff
@@ -291,6 +297,162 @@ Without an `event_time` query parameter, `/status_changes` shall return a `400 B
 
 [Top][toc]
 
+## Reports
+
+Reports are information that providers can send back to agencies containing aggregated data that is not contained within other MDS endpoints, like counts of special groups.
+
+The authenticated reports endpoint allows a user to pass in some parameters and get trip counts in the response.
+
+### Reports - Response
+
+**Endpoint:** `/reports`  
+**Method:** `GET`  
```

Comment:
Method is `GET` here, but down below the example shows `POST`.

Any reason this needs to be `POST`?

Reply:
No this probably should get Get. The POST is a left over.